addons/sale_project/models/project.py

```diff
@@ -65,7 +65,7 @@ def _check_sale_line_type(self):
             if task.sale_line_id:
                 if not task.sale_line_id.is_service or task.sale_line_id.is_expense:
                     raise ValidationError(_(
-                        'You cannot link the order item %(order_id)s - %(product_id)s to this task because it is a re-invoiced expense.',
+                        'You cannot link the order item %(order_id)s - %(product_name)s to this task because it is a re-invoiced expense.',
                         order_id=task.sale_line_id.order_id.id,
                         product_name=task.sale_line_id.product_id.name,
```

Comment:
could you change this part instead? this way we don't need to modify the translation

Reply:
Updated